readme.md

```diff
@@ -70,11 +66,4 @@ Basically the only tricky thing is how we handle custom entries in the SceneExpl
 Just take a look at the `Motion Event Pull Request`, which should've been added around `March 2016`, there you can see what was needed to add MotionEvents to the SDK.
 
 Other than that, we are more than happy to help, even if your addition is incomplete. Make sure you use the Netbeans formatting, obvious variable naming and commented and especially documented code, though.
-Before you open an issue, make sure that it is not an engine bug, but a real sdk bug. If you are uncertain whether it is a real bug or misuse, you can either open an issue or ask [on the hub](https://hub.jmonkeyengine.org) using the `Troubleshooting | jmonkeyplatform` category.
-
-## Issues when building
-__Could not find `jme3.1.0-stable`__ :  
-    When you occur this issue, you have to go back to the engine build step and see how the engine is called. Technically the engine version is collected out from git tags, or called SNAPSHOT. Then there are cases where this recognition fails. For example when the SDK has a tag but the engine is snapshot. Then take a look at `build.gradle` in the sdk repo. It contains a field called `ext.jmeVersion` or something, where you can specify the version.
-    
-__Could not find `jme3-jbullet`__ :  
-    jBullet is not part of jCenter, which means you forgot to build the engine first. See above.
+Before you open an issue, make sure that it is not an engine bug, but a real SDK bug. If you are uncertain whether it is a real bug or misuse, you can either open an issue or ask [on the hub](https://hub.jmonkeyengine.org) using the `Troubleshooting | jmonkeyplatform` category.
```

Comment:
"Before you open an issue...open an issue"? :thinking: It's not like the issue tracker is overloaded. Maybe just:
```suggestion
If you think you've encountered an issue with the SDK, please open an issue to let the developers know or post [on the hub](https://hub.jmonkeyengine.org) using the `Troubleshooting | jmonkeyplatform` category.
```